pkg/apis/operator/v1alpha1/types.go

```diff
@@ -84,8 +84,8 @@ type RuntimeCluster struct {
 
 // Ingress configures the Ingress specific settings of the runtime cluster.
 type Ingress struct {
-	// Domain specifies the IngressDomain of the cluster pointing to the ingress controller endpoint. It will be used
-	// to construct ingress URLs for system applications running in runtime cluster.
+	// Deprecated: This field is deprecated and will be removed soon. Please use `Domains` instead.
+	// TODO(scheererj): Drop this after v1.88 has been released.
```

Comment:
Should we wait 3 releases, so 'after v1.90'?

Reply:
Changed the comment to 'after v1.90' as suggested with https://github.com/gardener/gardener/pull/9038/commits/e623c777f0a6fa66b208296f382d5700906b4ee6.